### PR TITLE
RISDEV-3932 implement data extraction from xml

### PR DESCRIPTION
**Pull request by malte-laukoetter**

### CODE REVIEW

File: frontend/src/services/ldmldeModService.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  const temporalGroupNode = getNodeByEid(xml, temporalGroupEid)
+  if (!temporalGroupNode) return null
+
+  const eventRefEid = getStartEventRefEid(temporalGroupNode)
+  if (!eventRefEid) return null
+
+  const eventRefNode = getNodeByEid(xml, eventRefEid)
+  if (!eventRefNode) return null
+
+  return getEventRefDate(eventRefNode)
```
Comment (reckseba) [inline frontend/src/services/ldmldeModService.ts:61]: @malte-laukoetter why not checking here as well?

File: frontend/src/services/ldmldeModService.spec.ts [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+                    <akn:timeInterval GUID="ca9f53aa-d374-4bec-aca3-fff4e3485179" eId="meta-1_geltzeiten-1_geltungszeitgr-1_gelzeitintervall-1" refersTo="geltungszeit" start="#meta-1_lebzykl-1_ereignis-2"/>
+                 </akn:temporalGroup>
+              </akn:temporalData>
+            </akn:meta>
+          </akn:act>
+        </akn:akomaNtoso>
+      `)
+
+      expect(
+        getTimeBoundaryDate(
```
Comment (reckseba) [inline frontend/src/services/ldmldeModService.spec.ts:119]: Shall we also test the branches where getTimeBoundaryDate returns null?

Comment (malte-laukoetter) [reply frontend/src/services/ldmldeModService.spec.ts:119]: We could but as they are quite trivial I didn't see the point in it ^^

File: frontend/src/services/ldmldeModService.ts (hunk 1 above)
Comment (malte-laukoetter) [reply frontend/src/services/ldmldeModService.ts:61]: It returns null (or maybe undefined I'm not 100% sure) either way so it would change (basically?) nothing

**Comment by malte-laukoetter** [comment]
Thanks for looking over it :)

File: frontend/src/services/ldmldeModService.ts (hunk 1 above)
Comment (reckseba) [reply frontend/src/services/ldmldeModService.ts:61]: true

File: frontend/src/services/ldmldeModService.spec.ts (hunk 2 above)
Comment (reckseba) [reply frontend/src/services/ldmldeModService.spec.ts:119]: Maybe test-coverage but not quite sure if sonar is calculating on that
